policyengine_us/parameters/gov/territories/pr/tax/income/tax_rate/rate.yaml

```diff
@@ -0,0 +1,33 @@
+description: Puerto Rico taxes income at these rates for income over the threshold for the bracket. 
+brackets:
+  - threshold: 
+      2012-12-31: 0
+    amount:
+      2012-12-31: 0
+  - threshold:
+      2012-12-31: 9_000
+    amount:
+      2012-12-31: 0.07
+  - threshold:
+      2012-12-31: 25_000
+    amount:
+      2012-12-31: 0.14
+  - threshold:
+      2012-12-31: 41_500
+    amount:
+      2012-12-31: 0.25
+  - threshold:
+      2012-12-31: 61_500
+    amount:
+      2012-12-31: 0.33
+metadata:
+  amount_unit: /1
+  threshold_unit: currency_USD
+  type: single_amount # marginal_rate ??
```

Comment:
this should be a marginal rate - then we won't need the base tax as it will be mathematically derived